test/core_test.cpp

```diff
@@ -4,6 +4,7 @@
 // https://www.boost.org/LICENSE_1_0.txt
 
 #include "yk/core/type_traits.hpp"
+#include "yk/core/hash.hpp"
```

Comment:
I think this is not needed (I think the original error on CI was false-positive; implicit instantiation of unused template)